go.mod

```diff
@@ -75,3 +75,5 @@ require (
 	google.golang.org/protobuf v1.33.0 // indirect
 	gopkg.in/ini.v1 v1.67.0 // indirect
 )
+
+replace github.com/heimweh/go-pagerduty => github.com/imjaroiswebdev/go-pagerduty v0.0.0-20240419180041-eace2b31e15b
```

Comment:
This replace must be dropped after https://github.com/heimweh/go-pagerduty/pull/150 gets merged and `heimweh/go-pagerduty` should be updated pointing to latest version.

Reply:
Addressed in https://github.com/PagerDuty/terraform-provider-pagerduty/pull/861/commits/fb99d1928ad2220d485105b5a245c3dec755be2c.